test/skimNtuple_HHbtag.cpp

```diff
@@ -2959,28 +2957,32 @@ int main (int argc, char** argv)
 				  float lep1_thresh = trgAssigner.getLepton1PtThresh(PERIOD, "mutau");
 				  float lep2_thresh = trgAssigner.getLepton2PtThresh(PERIOD, "mutau");
 				  if (tlv_firstLepton.Pt()  < lep1_thresh) passSingle = 0;
-				  if (tlv_secondLepton.Pt() < lep2_thresh) passCross = 0;
-
-				  //lepton trigger
-				  double SFL_Data = muTrgSF->get_EfficiencyData(tlv_firstLepton.Pt(), fabs(tlv_firstLepton.Eta()));
-				  double SFL_MC   = muTrgSF->get_EfficiencyMC(tlv_firstLepton.Pt(), fabs(tlv_firstLepton.Eta()));
-
-				  double SFL_Data_Err = muTrgSF->get_EfficiencyDataError(tlv_firstLepton.Pt(), fabs(tlv_firstLepton.Eta()));
-				  double SFL_MC_Err   = muTrgSF->get_EfficiencyMCError(tlv_firstLepton.Pt(), fabs(tlv_firstLepton.Eta()));
-
+				  if (tlv_secondLepton.Pt() < lep2_thresh) passCross  = 0;
+
+				  // lepton trigger
+				  double SFL_Data = 1., SFL_MC = 1., SFL_Data_Err = 1., SFL_MC_Err = 1.;
```

Comment:
Following logic of some of the changes below, `SF_*_Err` should be initialized to 0